src/components/views/elements/AppTile.tsx

```diff
@@ -226,24 +253,14 @@ export default class AppTile extends React.Component<IProps, IState> {
         // Widget action listeners
         if (this.dispatcherRef) dis.unregister(this.dispatcherRef);
 
-        // if it's not remaining on screen, get rid of the PersistedElement container
-        if (!ActiveWidgetStore.instance.getWidgetPersistence(this.props.app.id)) {
-            ActiveWidgetStore.instance.destroyPersistentWidget(this.props.app.id);
-            PersistedElement.destroyElement(this.persistKey);
-        }
```

Comment:
this is the main change that fixes the issue. Instead of destroying it here we destroy it if the widget is not visible on screen anymore of if we switch room.
(this pr now fixes other bugs so i though it would be good to highlight this with a comment)